src/components/ReportActionItem/TaskView.js

```diff
@@ -52,54 +53,58 @@ function TaskView(props) {
     const disableState = !canEdit || !isOpen;
     return (
         <View>
-            <PressableWithSecondaryInteraction
-                onPress={Session.checkIfActionIsAllowed((e) => {
-                    if (e && e.type === 'click') {
-                        e.currentTarget.blur();
-                    }
+            <Hoverable>
```

Comment:
Can we have the `Hovered` component as a second child wrapped by `PressableWithSecondaryInteraction` like in [this example](https://github.com/allroundexperts/Expensify/blob/18841da7af6cf5c5f86ae3140020b1b6c12b91cf/src/pages/home/report/ReportActionItem.js#L482)?

Reply:
That's clear, thanks!